www/artists/arocamora/vessels.html

```diff
@@ -4,77 +4,104 @@
                 <br>
-                cactus blossom bottle: porcelain sculpture, under glazes, 1997
-            </p>
-            <p>
-                <a href="images/vessel02.jpg"><img src="images/vessel02.jpg" alt="vessel"></a>
                 <br>
-                VESSEL I: porcelain with underglazes
-            </p>
-            <p>
-                <a href="images/vessel01.jpg"><img src="images/vessel01.jpg" alt="Vessel"></a>
+                Anita Rocamora is visual artist who works in ceramics and mixed media. Anita uses natuer and humanity for insperation in her art work. Anita uses deferent shapes from natuer and put them in a shape for a vase. Anita trys to create objects that are entirely imaginary yet evoke a feeling of familiarity and recognition. In nature. Form and function flow from one another in even the most absurdly extravagant creations. The focus of her work is to exploration other shapes that no one ever used. Anitas primary materials are porcelian and stoneware. Clay contains traces of all that has existed on this planet. From single-cell organisms to mountains. To ground together by erosion and geological forces to create a material that can be transformed at will. Anita likes to make with these material objects for facilitate joy, awareness, whimsy, and a sense of connection to the earth.
+            </div>
+            <div id="footer">
+                Made by Derek Cameron 2008, updated by Anna Tunison 2010, updated by Taylor Petrychyn 2012
                 <br>
-                FLOATING VESSEL: porcelain with underglazes
-            </p>
-            <p>
-                <a href="images/blurry1.jpg"><img src="images/blurry1.jpg" alt="Blurry" width="300"></a>
+                &copy; Anita Rocamora
                 <br>
-                <a href="images/blurry2.jpg"><img src="images/blurry2.jpg" alt="Blurry2" width="300"></a>
-            </p>
-            <p>
-                <img src="images/slit1.jpg" onclick="this.src='images/slit2.jpg';" width="400" alt="no alt text">
-                <br>
-                Meacham: Clay, metal, glaze, 37 x 33 x 29cm, Repose: 2011
-            </p>
-        </article>
-        <section>
-          <ul>
-              <li>
-                  <a href="../">Sask Artists</a>
-              </li>
-              <li>
-                  <a href="vessels.html">Vessels</a>
-              </li>
-              <li>
-                  <a href="figures.html">Figures</a>
-              </li>
-              <li>
-                  <a href="index.html">Home</a>
-              </li>
-              <li>
-                  <a href="sculpture.html">Sculpture</a>
-              </li>
-          </ul>
-        </section>
-        <footer>
-            <p>
-                Created by Anna Tunison 2010
-            </p>
-        </footer>
+                <img src="images/anita_roca.jpg" alt="Anita Rocamora">
```

Comment:
Looking at the sculpture page, these images at the bottom are probably unnecessary. I can understand having it on the index page, which is the closest to a bio on this site, but the gallery pages probably don't need a picture of the artist.